Packs/Office365AndAzureAuditLog/Integrations/MicrosoftPolicyAndComplianceAuditLog/MicrosoftPolicyAndComplianceAuditLog.yml

```diff
@@ -161,7 +161,7 @@ script:
   runonce: false
   script: '-'
   type: powershell
-  dockerimage: demisto/powershell-ubuntu:7.3.0.49844
+  dockerimage: demisto/pwsh-exchangev3:1.0.0.49863
```

Comment:
why the docker switch?

Reply:
This is expected with the library change.